### DISCUSSION TITLE
Playing with permission

### DISCUSSION THREAD

**Discussion by 6-ril** [kimai/kimai#2597] [Questions & Answers] [answered]
Hello, 
I'm trying to give the ability to create customer, project and activity to Team_Leader.
So I add this in local.yaml :
permissions :
  roles :
    ROLE_TEAMLEAD :['create_invoice','create_activity','edit_activity','delete_activity',‘create_project’,’edit_project’,’delete_project’,’create_customer’,’edit_customer’,’delete_customer’]

Now a team leader can create a customer and an activity but when creating a project, I have an error message similar to "missing authorization, redirecting to log in ?"

There's probably some permissions missing ...
Any idea of which one ?

Thanks, 
Cyril

**Comment by kevinpapst** [comment]
Completely revert your changes in local.yaml and use the `Roles` administration.

Afterwards report back!

**Reply by 6-ril** [reply]
oh ... ok 
Thanks

**Comment by 6-ril** [comment, marked as answer]
I took me time to find where was this role administration ... :-D

So to make Team_Lead able to : 
- manage team activity, I select create_activity and delete_activity in the activity (admin) section
- manage team project, I select create_project, delete_project and view_project in the project (admin) section
- manage team customer, I select create_customer, delete_customer and view_customer in the customer (admin) section

I had to add view_project and view_customer otherwise, the error message still appear even if the project is created.

**Reply by 6-ril** [reply]
Thanks KevinPapst

**Reply by kevinpapst** [reply]
Hahah, very good. In the latest version, the Roles administration is directly in the side menu.

**Reply by 6-ril** [reply]
I'm not using the last version, probably because I'm using YunoHost as "server manager".
